app/services/item_create_service.rb

```diff
@@ -7,19 +7,9 @@ def initialize(organization_id:, item_params:, request_unit_ids: [])
 
   def call
     new_item = organization.items.new(item_params)
-    organization.transaction do
-      new_item.save!
-      if Flipper.enabled?(:enable_packs)
-        new_item.sync_request_units!(@request_unit_ids)
-      end
-
-      organization.storage_locations.each do |sl|
-        InventoryItem.create!(
-          storage_location_id: sl.id,
-          item_id: new_item.id,
-          quantity: 0
-        )
-      end
+    new_item.save!
```

Comment:
Lost the transaction, though I guess if this is only doing a single db update then it doesn't matter